public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -303,6 +291,23 @@ export function createSceneVariableFromVariableModel(variable: TypedVariableMode
     name: variable.name,
     label: variable.label,
   };
+  if (variable.type === 'adhoc') {
+    return new AdHocFiltersVariable({
```

Comment:
I don't use `AdHocFiltersVariable .create` because I cannot pass the `label`, `description`, `skipUrlSync` and `hide`